src/rules/no-entire-object-hook-deps.ts

```
@@ -295,10 +314,79 @@ function getObjectUsagesInHook(
+
+      // Check if this member expression involves our target object
+      let current: TSESTree.Node = memberExpr;
+      let foundTargetObject = false;
+      let hasDynamicComputed = false;
+
+      // Walk up the member expression chain to see if it involves our target object
+      while (current.type === AST_NODE_TYPES.MemberExpression) {
+        const currentMember = current as TSESTree.MemberExpression;
+
+        // Check if this level uses dynamic computed property access
+        if (currentMember.computed &&
```

ESLint

[error] Insert `⏎··········`

(prettier/prettier)